src/main/java/io/github/cottonmc/cotton/gui/widget/WLabel.java

```diff
@@ -176,6 +180,28 @@ public WLabel setColor(int color, int darkmodeColor) {
 		return this;
 	}
 
+	/**
+	 * Checks whether shadows should be drawn for this label.
+	 * 
+	 * @return {@code true} shadows should be drawn, {@code false} otherwise
+	 * @since 11.0.1
+	 */
+	public boolean areShadowsDrawn() {
```

Comment:
`getDrawShadows` would be a better name since it's a getter (this will give property syntax for Kotlin modders).